manifest.json

```diff
@@ -20,7 +20,7 @@
   ],
   "content_scripts": [{
     "matches": ["http://*/*", "https://*/*"],
-    "js": ["hover-styles.js", "line-spacing.js", "ruler.js", "split-paragraphs.js", "update-color.js", "background-images.js", "font.js","script.js"],
+    "js": ["hover-styles.js", "line-spacing.js", "ruler.js", "split-paragraphs.js", "update-color.js", "background-images.js", "font.js",  "palettes.js, "script.js"],
```

Comment:
It looks like you removed the main "script.js" file. Also, the comma is inside the "palettes.js" string

Reply:
Forgot the closing " at the end of palettes.js which is what caused it - fixed as our conversations overlapped